app/cards/Card.tsx

```diff
@@ -37,7 +43,7 @@ const config = {
 };
 
 @graphql(cardDataFromId, config)
-export default class Card extends React.PureComponent<Props, {}> {
+export default class Card extends React.Component<Props, {}> {
```

Comment:
Just an FYI -- TypeScript now has the ability to set default generic parameters and the React type definitions now have them set appropriately. 

So instead of `React.Component<Props, {}> {` you can simply write `React.Component<Props> {`.

If you have no props or state, you can now write just `React.Component {`